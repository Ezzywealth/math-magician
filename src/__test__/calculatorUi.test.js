/* eslint-disable no-undef */
import * as React from 'react'
import Calculator from "../components/calculatorUi";
import renderer from 'react-test-renderer';

test('shows the children when the checkbox is checked', () => {
  const tree = renderer
  .create(<Calculator/>)
  .toJSON();
  expect(tree).toMatchSnapshot();

})