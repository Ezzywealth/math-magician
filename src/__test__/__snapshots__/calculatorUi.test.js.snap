// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows the children when the checkbox is checked 1`] = `
<div
  className="container"
>
  <h1
    className="calc_header"
  >
    Lets do some Math
  </h1>
  <div
    className="calculator_container"
  >
    <div
      className="input_field"
    >
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="row"
    >
      <button
        className=""
        onClick={[Function]}
      >
        AC
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        +/-
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        %
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        /
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        7
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        8
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        9
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        *
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        4
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        5
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        6
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        -
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        1
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        2
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        3
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        +
      </button>
      <button
        className="zero"
        onClick={[Function]}
      >
        0
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        .
      </button>
      <button
        className=""
        onClick={[Function]}
      >
        =
      </button>
    </div>
  </div>
</div>
`;
